frontend/app/root.tsx

```diff
@@ -58,7 +58,7 @@ function PageTitle() {
     .filter((handle): handle is RouteHandle => handle !== undefined)
     .filter((handle) => handle.pageTitlei18nKey !== undefined)
     .map((handle) => handle.pageTitlei18nKey)
-    .reduce((last, curr) => curr ?? last);
+    .pop();
```

Comment:
This isn't functionally the same. If the last element is `undefined`, then `pop()` will return `undefined`.

`.reduce((last, curr) => curr ?? last)` will return the last non-`undefined` element, unless they're all undefined.

Reply:
You're right. Sorry for my confusion. Thanks :)